.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: 2023 Dennis Gläser <dennis.glaeser@iws.uni-stuttgart.de>
+# SPDX-License-Identifier: MIT
+
+name: releases
+
+on:
```

Comment:
what's the desired workflow? I update all files with versions, merge to main, and then whenever (possibly much later) I push the tag, the automated release happens?

Reply:
Do we need to restrict the pipeline to main?

Reply:
it's only triggered for tags that start with "v" anyway, so no branches at all. Should be fine I guess?

Reply:
I could add a paragraph on this to the main readme in the development section

Reply:
I added a paragraph with the envisioned workflow

Reply:
I made some modifications to the script & adjusted the readme. From my side this is fine.